### ISSUE TITLE
Does yolov5 have some way to detect single type object?

### ISSUE THREAD

**Issue by abczww** [ultralytics/yolov5#726]
## ❔Question
I am using yolov5 to detect objects by webcam, yolov5 could detect so many type objects, but I don't care about most type of them, I just need single type, like: person. I think if yolov5 just detect single type object it should work faster. But I didn't find a way to impleent this requirement. I have to add some extra codes to filter the result to get all persons which means the project became slower theoretically.

In a nutshell, I want to yolov model have many types and could detect all types objects, and it should could detect some specified type objects or single type object either.
e.g.:
I trained many types in model including: 'person', 'bicycle', 'car', 'motorcycle', 'airplane', 'bus', 'train', 'truck', 'boat', 'traffic light'
But I just want to detect 'person', 'bicycle',
Or just 'person',

**Comment by glenn-jocher** [comment]
@abczww detect.py can filter detections by class:
https://github.com/ultralytics/yolov5/blob/1ddf692bf89ac47d5e2dc8333cd0f4f145bb7f61/detect.py#L160

To detect only people in images:
`python detect.py --classes 0`

**Comment by jomdolalas** [comment]
@abczww Hi! Would like to know how you were able to use your webcam as the input? The code
!python detect.py --source 0 
does not work for me. I am using a laptop with a webcam installed. Do I need to use an external one?

**Comment by Frank1126lin** [comment]
> @abczww Hi! Would like to know how you were able to use your webcam as the input? The code
> !python detect.py --source 0
> does not work for me. I am using a laptop with a webcam installed. Do I need to use an external one?

@jomdolalas just use `--source 0` or `--source 1`  etc is ok.

**Comment by mintu07ruet** [comment]
Hi @ glenn-jocher 
I was try to detect just bicycle using python detect.py --classes x
what is the labels number for bicycle. I tried to find it, in coco data I saw it is 2, but it 2 is not working for bicycle, 2 is actually for car. Thanks!

**Comment by TuyenSD** [comment]
i want to detect just 1 biggest object, so what shoud i do, thks for listening

**Comment by glenn-jocher** [comment]
@TuyenSD simply select the largest detection.

**Comment by mintu07ruet** [comment]
@glenn-jocher ,
I was using yolov5+deep tracker (https://github.com/mikel-brostrom/Yolov5_DeepSort_Pytorch) to track the following objects-
0= Person
1= Bicycle
3= motorcycle
7=truck
17=Horse
36= Skateboard
Although car is tracking properly but frame is not hold continuously for bicycle and pedestrian. Any suggestions to improve the tracking! Here is the output video https://drive.google.com/file/d/16ZfNxG8p0dEpZZJ9I2vZ__jGJ4IT8w2I/view?usp=sharing
Thanks!
